ntfy/cli.py

```diff
@@ -26,6 +27,11 @@
 except ImportError:
     psutil = None
 
+try:
+    import argcomplete
```

Comment:
So you have to manually install argcomplete? I'm not really opposed to making it a dependency. But at the very least it shoul dbe an optional extra dependency. Ie: `pip install ntfy[autocomplete]`, to do that, add `'autocomplete':['argcomplete'],` to this dict: https://github.com/dschep/ntfy/blob/master/setup.py#L8

Reply:
Sure, updated with README comment as well.